internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestHybridStoreRepartitioningWithMultiDataCenter.java

```diff
@@ -159,12 +191,23 @@ public void testHybridStoreRepartitioning() {
         .setHybridOffsetLagThreshold(1000);
     TestWriteUtils.updateStore(storeName, parentControllerClient, updateStoreParams);
 
+    for (ControllerClient controllerClient: childControllerClients) {
+      TestUtils.waitForNonDeterministicAssertion(30, TimeUnit.SECONDS, () -> {
+        StoreInfo storeInfo = controllerClient.getStore(storeName).getStore();
+        Assert.assertEquals(storeInfo.getLargestUsedRTVersionNumber(), 1);
```

Comment:
Let's also verify the RT names in the store level hybrid config after 1st time changing to hybrid.

Reply:
adding